app/org/sagebionetworks/bridge/models/notifications/NotificationMessage.java

```diff
@@ -0,0 +1,81 @@
+package org.sagebionetworks.bridge.models.notifications;
+
+import java.util.Objects;
+
+import org.sagebionetworks.bridge.models.BridgeEntity;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+
+/**
+ * For information on the push notification JSON structures that are sent to SNS, see:
+ * http://docs.aws.amazon.com/sns/latest/dg/mobile-push-send-custommessage.html
+ * 
+ * Note: There are many more fields that we can and will provide once we have an end-to-end system working. So we have
+ * introduced a builder in anticipation of expanding this model.
+ */
+@JsonDeserialize(builder=NotificationMessage.Builder.class)
+public final class NotificationMessage implements BridgeEntity {
+
+    /**
+     * Very short rendition of the notification. For example, this value will be displayed on an Apple iWatch, where the
+     * message is shown for a short time only.
+     */
+    private final String subject;
+    
+    /**
+     * Full notification message. 
+     */
+    private final String message;
```

Comment:
This really depends on the platform, and I noticed that using the simple interface for sending messages, SNS did not add the subject to the JSON it produced. But there's a field that maps to this in both APNS and GCM so when I move to constructing the JSON myself, it should be used (but again I'm not 100% sure how. I'll update the docs as we learn more).